project/src/main/webapp/script.js

```diff
@@ -0,0 +1,85 @@
+var map;
+var autocomplete;
+var photoURLS; 
+var image;
+
+function execute() {
+    initMap();
+    search();
+    getCatalog();
+}
+
+function initMap() {
+    map = new google.maps.Map(document.getElementById("map"), {
+        center: { lat: -33.866, lng: 151.196 },
+        zoom: 15
+    });
+}
+
+function search() {
```

Comment:
Didn't I take on search last week, is this an additional piece for autocomplete?